main.nf

```diff
@@ -147,7 +148,8 @@ process fastqc {
     publishDir "${params.outdir}/fastqc", mode: 'copy'
 
     input:
-    set val(name), file(reads:'*') from fastQC_input
+    set val(name) from name_for_fastqc
```

Comment:
Is this only used for the `tag "$name"` line? I think that this may work with `tag "$reads"` now that it's SE only. May be able to remove all of these lines and all of the channels that are used for them.